lib/rubocop/cop/rails/http_positional_arguments.rb

```diff
@@ -0,0 +1,89 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop is used to identify usages of `http methods
+      # like get,post,put,path` without the usage of keyword arguments
+      # in your tests and change them to use keyword arguments.
+      #
+      # @example
+      #   # bad
+      #   get :new, { user_id: 1}
+      #
+      #   # good
+      #   get :new, params: { user_id: 1 }
+      class HttpPositionalArguments < Cop
+        MSG = 'Use `keyword arguments` instead of ' \
+              'positional arguments for http call: '.freeze
+        KEY_WORD_ARGS = [:headers, :env, :params, :body, :flash, :as].freeze
+        HTTP_METHODS = [:get, :post, :put, :patch, :delete, :head].freeze
+
+        def on_send(node)
+          receiver, http_method, controller_node, data = *node
+          # if the first word on the line is not an http method, then skip
+          return unless HTTP_METHODS.include?(http_method)
+          # if the data is nil then we don't need to add keyword arguments
+          # because there is no data to put in params or headers, so skip
+          return if data.nil?
+          return unless needs_conversion?(data)
+          # ensures this is the first method on the line
+          # there is an edge case here where sometimes the http method is
+          # wrapped into another method, but its just safer to skip those
+          # cases and process manually
+          return unless receiver.nil?
+          # a http_method without a path?, must be something else
+          return if controller_node.nil?
+          add_offense(node, node.loc.selector, "#{MSG}#{node.method_name}.")
```

Comment:
Instead of using interpolation you can use `format` and `%s` in the original message. That's our common approach for constructing messages.